vibration/notifications.html

```diff
@@ -0,0 +1,27 @@
+---
+feature_name: NotificationOptions.vibrate
+chrome_version: 45
+feature_id: 5727996321202176
+---
+
+<p>When displaying a system notification your device may vibrate to alert the user. You can specify the vibration pattern that should be used in the same way as for the <a href="./index.html">Vibration API</a>.</p>
+
+<p>When creating the notification, just set the 'vibrate' property of the notification options to be a Vibration API pattern.</p>
+
```

Comment:
This is a side-effect of us not supporting "new Notification()" on Android, which iOS and Firefox OS do support. Such notifications can support vibration just as well.